### ISSUE TITLE
react-axe 3.0.0-alpha.1 + axe 3.0.0-beta.2 errors

### ISSUE THREAD

**Issue by MrHavard** [dequelabs/react-axe#28]
I was running react-axe 3.0.0-alpha.1 which originally included axe 3.0.0-beta.1 as a dependency. I did a fresh install on a project today and see that the product is now throwing nothing but errors to the console. I looked and the only major difference I can find is that the axe dependency is now 3.0.0-beta.2. 

The error I'm getting is 

```
undefined
axe.js?3204:2647 Uncaught (in promise) TypeError: Cannot read property 'actualNode' of undefined
    at _loop (axe.js?3204:2647)
    at Object.axe.utils.getSelectorData (axe.js?3204:2687)
    at generateSelector (axe.js?3204:2792)
    at Object.createUniqueSelector [as getSelector] (axe.js?3204:2848)
    at DqElement.get selector [as selector] (axe.js?3204:2256)
    at eval (axe.js?3204:1228)
    at Array.map (<anonymous>)
    at eval (axe.js?3204:1221)
    at Array.map (<anonymous>)
    at eval (axe.js?3204:1218)
```
It will not produce any valid responses in the console and nothing has changed between the two copies of my project (one working with beta.1 and one not working with beta.2)

**Comment by WilcoFiers** [comment]
OK, I think I found the issue: https://github.com/dequelabs/react-axe/blame/develop/index.js#L64

`axeCore._audit.data.failureSummaries[key].failureMessage()` throws because it's called after `axe.run()` already cleaned up the flat tree. I'll see what I can do about that.

**Comment by WilcoFiers** [comment]
We found the cause of the issues and will include the fix in the 3.0 release.
